registry/middleware/registry/middleware.go

```diff
@@ -38,3 +40,14 @@ func Get(ctx context.Context, name string, options map[string]interface{}, regis
 
 	return nil, fmt.Errorf("no registry middleware registered with name: %s", name)
 }
+
+// RegisterOptions adds more options to RegistryOption list.
```

Comment:
I find it worth noting that these options get applied before any other configuration-based options.